cmd/ef_tests/levm/report.rs

```diff
@@ -353,6 +357,69 @@ impl fmt::Display for AccountUpdatesReport {
         for levm_updated_account_only in self.levm_updated_accounts_only.iter() {
             writeln!(f, "  {levm_updated_account_only:#x}:")?;
             writeln!(f, "{}", "    Was updated in LEVM but not in REVM".red())?;
+            let initial_account = self
+                .initial_accounts
+                .get(levm_updated_account_only)
+                .unwrap();
+            let updated_account_update = self
+                .levm_account_updates
+                .iter()
+                .find(|account_update| &account_update.address == levm_updated_account_only)
+                .unwrap();
+            let updated_account_storage = updated_account_update
+                .added_storage
+                .iter()
+                .map(|(key, value)| {
+                    let storage_slot = StorageSlot {
+                        original_value: *value,
+                        current_value: *value,
+                    };
+                    (*key, storage_slot)
+                })
+                .collect();
+            let updated_account_info = updated_account_update.info.clone().unwrap();
+            let updated_account = Account::new(
+                updated_account_info.balance,
+                updated_account_update.code.clone().unwrap_or_default(),
+                updated_account_info.nonce,
+                updated_account_storage,
+            );
+            if initial_account.info.balance != updated_account.info.balance {
+                writeln!(
+                    f,
+                    "{}",
+                    format!(
+                        "      Balance updated: {initial_balance} -> {updated_balance}",
+                        initial_balance = initial_account.info.balance,
+                        updated_balance = updated_account.info.balance
+                    )
+                    .red()
+                )?;
+            } else if initial_account.info.nonce != updated_account.info.nonce {
+                writeln!(
+                    f,
+                    "{}",
+                    format!(
+                        "      Nonce updated: {initial_nonce} -> {updated_nonce}",
+                        initial_nonce = initial_account.info.nonce,
+                        updated_nonce = updated_account.info.nonce
+                    )
+                    .red()
+                )?;
+            } else if initial_account.info.bytecode != updated_account.info.bytecode {
+                writeln!(
+                    f,
+                    "{}",
+                    format!(
+                        "      Code updated: {initial_code}, {updated_code}",
+                        initial_code = hex::encode(&initial_account.info.bytecode),
+                        updated_code = hex::encode(&updated_account.info.bytecode)
+                    )
+                    .red()
+                )?;
+            } else {
+                writeln!(f, "{}", "      No changes".green())?;
```

Comment:
If storage was the thing that changed maybe we shouldn't be saying that there were No changes. Maybe it would be good to add storage comparison too to the report.

Reply:
Yes! Done in [987e095](https://github.com/lambdaclass/ethrex/pull/1280/commits/987e09501d84b05ae6aa83a188d927f34ad60b8f).